index.js

```diff
@@ -0,0 +1,114 @@
+  // On app load, get all tasks from localStorage
+  window.onload = loadTasks;
+
+
+
+  /* function loadTasks() {
+    // check if localStorage has any tasks
+    // if not then return
+    if (localStorage.getItem("tasks") == null) return;
+
+    // Get the tasks from localStorage and convert it to an array
+    let tasks = Array.from(JSON.parse(localStorage.getItem("tasks")));
+
+    // Loop through the tasks and add them to the list
+    tasks.forEach(task => {
+      const list = document.querySelector("ul");
+      const li = document.createElement("li");
+      li.innerHTML = `<input type="checkbox" onclick="taskComplete(this)" class="check" ${task.completed ? 'checked' : ''}>
+        <input type="text" value="${task.task}" class="task ${task.completed ? 'completed' : ''}" onfocus="getCurrentTask(this)" onblur="editTask(this)">
+        <i class="fa fa-trash" onclick="removeTask(this)"></i>`;
+      list.insertBefore(li, list.children[0]);
+    });
+  } */
+
+  function addTask() {
+    const task = document.querySelector("form input");
+    const list = document.querySelector("ul");
+    // return if task is empty
+    if (task.value === "") {
+      alert("Please add some task!");
+      return false;
+    }
+    // check is task already exist
+    if (document.querySelector(`input[value="${task.value}"]`)) {
+      alert("Task already exist!");
+      return false;
+    }
+
+    // add task to local storage
+    localStorage.setItem("tasks", JSON.stringify([...JSON.parse(localStorage.getItem("tasks") || "[]"), { task: task.value, completed: false }]));
+
+    // create list item, add innerHTML and append to ul
+    const li = document.createElement("li");
+    li.innerHTML = `<input type="checkbox" onclick="taskComplete(this)" class="check">
+    <input type="text" value="${task.value}" class="task" onfocus="getCurrentTask(this)" onblur="editTask(this)">
+    <i class="fa fa-trash" onclick="removeTask(this)"></i>`;
+    list.appendChild(li);
+    // clear input
+    task.value = "";
+  }
+
+    // On form submit add task
+    document.querySelector("form").addEventListener("submit", e => {
+        e.preventDefault();
+        addTask();
+      });
+
+  function taskComplete(event) {
+    let tasks = Array.from(JSON.parse(localStorage.getItem("tasks")));
+    tasks.forEach(task => {
+      if (task.task === event.nextElementSibling.value) {
+        task.completed = !task.completed;
+      }
+    });
```

Comment:
[OPTIONAL]  According to the best JavaScript practice, I would advise you to use an arrow function instead of the normal function
you can peruse this [Link](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) to learn more about the best practice in JavaScript.

I think you can make your code look something like this; 👇🏽
```
const taskComplete = (event) => {

}
```

I understand that this is a personal tutorial project, but I'd like to kindly suggest that if you have the time, you can implement this so your code follows the requirements. 🤗 🤗